pages/4_📋_Review_Perfect_Simulation.py

```diff
@@ -65,7 +66,7 @@ def app():
 
         number_of_players = st.slider("Select the number of top players to show:", min_value=0,
                                       max_value=len(perfect_simulator_df.index), value=30)
-
+        perfect_simulator_df.to_csv("data/player_rewards.csv")
```

Comment:
should annotate the generated file name so that we know which tournament-season's rewards are being saved.

Reply:
hey @gireesh-fractal this line is actually not required, I have removed it.. thanks for spotting it